backend/store/etcd/environment_store_test.go

```diff
@@ -71,3 +67,38 @@ func TestEnvStorage(t *testing.T) {
 		assert.Equal(t, 1, len(envs))
 	})
 }
+
+func TestDeleteEnvironment_GH1567(t *testing.T) {
```

Comment:
Perhaps I'm missing some context here, but what does `GH1567` mean?

Reply:
That's the issue this test was written to reproduce. (GH stands for Github)

Reply:
Ah very cool!